packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -113,7 +113,7 @@ describe("Data Store Context Tests", () => {
 				0,
 				0,
 			);
-			summarizerNode.startSummary(0, createChildLogger());
+			summarizerNode.startSummary(0, createChildLogger(), 0);
```

Comment:
Seems odd to pass the same value (`0`) for both these sequence numbers.  Even if it doesn't "matter" in the test execution, might as well do 1 and 0 or something instead.

Reply:
0 is correct here for both numbers. Reference sequence number is the number off ops processed, and since 0 ops were processed, then the value is 0. 0 also happens to be the sequence number of the last summary (there was none)